Source/Core/DolphinQt/NetPlay/NetPlayBrowser.cpp

```diff
@@ -278,6 +271,17 @@ void NetPlayBrowser::OnSelectionChanged()
       ->setEnabled(!m_table_widget->selectedItems().isEmpty());
 }
 
+void NetPlayBrowser::OnUpdateStatusRequested(const QString& status)
+{
+  m_status_label->setText(status);
+}
+
+void NetPlayBrowser::OnUpdateListRequested(const std::vector<NetPlaySession>& sessions)
```

Comment:
This can take the vector by value and then std::move `sessions` into `m_sessions`

Reply:
Done!